proposals/buildfile.md

```diff
@@ -0,0 +1,186 @@
+# Jib CLI Buildfile Specification
+
+Specification for a YAML buildfile describing building a container image. This buildfile can be
+used by the jib-cli to generate a container. It is translated directly into a buildplan and
+passed to the builder.
+
+```yaml
+apiVersion: jib/v0alpha1
+kind: Config
+
+# "FROM" with detail for manifest lists or multiple architectures
+from:
+  image: "gcr.io/distroless/java:8"
+  platforms:
+    - architecture: "arm"
+      os: "linux"
+      os.version: "a"
+      os.features:
```

Comment:
It would seem like `os.version` and `os.features` should be subitems of `os`:
```yaml
os:
  id: "linux"
  version: "a"
  features: ["b1", "b2"]
```

Reply:
yeah that's just how they're represented in the specs, so I was mirroring them. not sure what the right thing in here, leaning to keep os.version

Reply:
https://docs.docker.com/registry/spec/manifest-v2-2/#manifest-list-field-descriptions

Reply:
added section about behavior for builds with multiple platforms